### ISSUE TITLE
LoadingController message does not accept HTML

### ISSUE THREAD

**Issue by russcarver** [ionic-team/ionic#15540]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)          : 4.1.1
   Ionic Framework            : @ionic/angular 4.0.0-beta.5
   @angular-devkit/core       : 0.7.5
   @angular-devkit/schematics : 0.7.5
   @angular/cli               : 6.1.5
   @ionic/ng-toolkit          : 1.0.7
   @ionic/schematics-angular  : 1.0.5

Cordova:

   cordova (Cordova CLI) : 7.1.0
   Cordova Platforms     : android 6.4.0
   Cordova Plugins       : cordova-plugin-ionic-webview 2.1.3, (and 6 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/rc101077/Library/Android/sdk)
   ios-deploy        : 2.0.0
   ios-sim           : 7.0.0
   NodeJS            : v8.11.4
   npm               : 6.4.1
   OS                : macOS High Sierra
   Xcode             : Xcode 9.4.1 Build version 9F2000
```

**Describe the Bug**
Per the notes of https://github.com/ionic-team/ionic/issues/15429 , the `create` method of the `LoadingController` now uses the `message` parameter in place of `content`.  The old `content` parameter used to take HTML and render it as such. Now, putting HTML in the `message` shows only the actual HTML markup and not the rendered version.

**Steps to Reproduce**
1. Use the following code:
```
private async showLoading(): Promise<void> {
    this.spinner = await this.loadingController.create({
      cssClass: 'ati-loading-spinner',
      duration: 3000,
      message: `<div style="font-weight: bold;">Test</div>`
    });
    await this.spinner.present();
}
```
2. This pops up a "spinner" that shows `<div style="font-weight: bold;">Test</div>` in it. 
3. It should show only "Test" in bold font.

**Comment by KevRyan2** [comment]
@paulstelzer this seems like more of putting back lost functionality than a feature request?  as an add-on to everything stated above, because the component was moved to the shadow DOM sometime around the release of rc0 , which restricted editable properties to only those provided css variables, it also removes the ability to make any sort of custom loader entirely now:
ex. As a "hack" to help us get around the loss of "content", we were using 'background-image' to display a gif, but that property has been removed now... do you have any suggestions in the meantime until either content gets put back or another solution is found?

**Comment by ocinbat** [comment]
Any progress on this one?

**Comment by clauabad** [comment]
Me too need it!

**Comment by manishdubeit** [comment]
Any update on this

**Comment by HansKrywaa** [comment]
I think this is implemented now?
https://github.com/ionic-team/ionic/blob/36f98eda1b68863ee262f5dffe72510465cd8f88/core/src/components/loading/loading.tsx#L204

Can somebody check if this is still a Issue?

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as this was fixed in Ionic Framework v4.4.1: https://github.com/ionic-team/ionic/pull/18242.
